module/zcommon/zfs_fletcher_sse.c

```diff
@@ -48,7 +48,7 @@
 #include <zfs_fletcher.h>
 
 struct zfs_fletcher_sse_array {
-	uint64_t v[2] __attribute__((aligned(16)));
+	uint64_t v[2];
```

Comment:
@rlaager The downsides are minimal here because the unaligned operations are only in .fini and the prelude to the loop where the vectorization does most of the work.

From what I can yearn, this problem was resolved in GCC 4.6.